src/palace.pyx

```diff
@@ -2338,10 +2338,10 @@ cdef class ChorusEffect(BaseEffect):
     """
     cdef alure.EFXCHORUSPROPERTIES properties
 
-    def __init__(self, waveform: str = 'triangle',
+    def __init__(self, waveform: int = 1,
```

Comment:
It was fine before lol

Reply:
Wut, it said that it needs an int before o.O